### DISCUSSION TITLE
Request a bottle? Find relevant logs?

### DISCUSSION THREAD

**Discussion by tobylane** [Homebrew/discussions#470] [Everyday usage]
What's the process for requesting a bottle of a specific formula? Maybe it'll happen anyway for big sur arm, but it seems we can also request it for linux?

Once the bottle is built how do we find the logs? I'm having major difficulty with the action job lists.

Please bottle `corsixth` for big sur arm and linux, and link to the logs.

**Comment by SMillerDev** [comment]
It's triggered by maintainers for ARM since we're slowly working out way through the remaining 30%. For Linux you'll have to make sure it builds, and then make an issue requesting a bottle.

**Comment by tobylane** [comment]
Thanks. Do you have a process for finding the logs of a randomly picked build in Actions?
